### ISSUE TITLE
[Screen Reader – BotKit – Header]: Role = “Navigation” and aria-label should be defined for header section.

### ISSUE THREAD

**Issue by v-gasikk** [howdyai/botkit-website#12]
**User Experience:**
Users who rely on screen readers will affect to navigate, if Role = “Navigation” and aria-label is not defined for header section.

**Test Environment:**
OS: Win 10 Version 1809 (OS Build 17763.316)
URL: https://botkit.ai/
Browser: Edge
Screen Reader: Narrator

**Repro Steps:**
Step 1: Open the above URL in Edge browser.
Step 2: Open developer tool bar (F 12)
Step 3: Navigate header section of the page.
Step 4: Verify whether Role = “Navigation” and aria-label is defined for header section.

**Actual Result:**
Role = “Navigation” and aria-label is not defined for the header section.

**Expected Result:**
Role = “Navigation” and aria-label should be defined for header section.

**Note**: For Left navigation pane aria-label should be defined.

**MAS Reference Link:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}](url)
![MAS1 3 1_BotKit_Main issue](https://user-images.githubusercontent.com/48272005/54472127-70630380-47e9-11e9-9b45-7fddbb6437f3.png)
![MAS1 3 1_BotKit_Note issue](https://user-images.githubusercontent.com/48272005/54472128-70630380-47e9-11e9-826e-f3a835bd0246.png)

**Comment by benbrown** [comment]
These attributes have been added.

**Comment by v-gasikk** [comment]
Issue is Partially Fixed.
Main issue is Fixed.
Note issue: For Left navigation pane aria-label should be defined - Not Fixed.

**Comment by benbrown** [comment]
@v-gasikk I have added an aria-label to the left nav.

**Comment by v-gasikk** [comment]
Verified. Issue is Fixed. Hence Closing the bug.
